Gemfile

```diff
@@ -37,11 +37,12 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
```

Comment:
- You have done well so far, but I have a [CORS error](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.contentstack.com/docs/developers/how-to-guides/understanding-and-resolving-cors-error&ved=2ahUKEwiqvPGy7rmEAxWZWEEAHVytDyMQFnoECDIQAQ&usg=AOvVaw1OcT8GndnBKh05K5EfdWFH) on your application, hence the greeting are not displayed on the front end. Please fix this by installing the `rack-cors` gem above (but currently commented out)
- Also in the `./config/initializers/cors.rb` directly, please ensure that your CORS file is properly configured. The file should look like this:
```ruby
    Rails.application.config.middleware.insert_before 0, Rack::Cors do
     allow do
        origins "*"

        resource "*",
        headers: :any,
        methods: [:get, :post, :put, :patch, :delete, :options, :head]
    end
  end
```